### PR TITLE
Integrate rofiFirmware into rmake ecosystem

### PR DESCRIPTION
**Pull request by yaqwsx**

### CODE REVIEW

File: releng/tools/_idf_monitor.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!python
```
Comment (ficap) [inline releng/tools/_idf_monitor.py:1]: this shebang does not seem to work with bash. I suggest to use something like `#!/usr/bin/env python`

File: releng/tools/rflash [hunk 2]
```diff
@@ -0,0 +1,45 @@
+#!python
```
Comment (ficap) [inline releng/tools/rflash:1]: this shebang does not seem to work with bash. I suggest to use something like `#!/usr/bin/env python3`

File: releng/tools/rmonitor [hunk 3]
```diff
@@ -0,0 +1,41 @@
+#!python
```
Comment (ficap) [inline releng/tools/rmonitor:1]: this shebang does not seem to work with bash. I suggest to use something like `#!/usr/bin/env python`

**Review by ficap — CHANGES_REQUESTED** [review]

File: examples/hal/wifi/partitions.csv [hunk 4]
```diff
@@ -1,5 +0,0 @@
-# ESP-IDF Partition Table
```
Comment (ficap) [inline examples/hal/wifi/partitions.csv:1]: this one will most probably be needed because final binary cannot fit into default 1M partition

**Review by Ondraceq — COMMENTED** [review]
Musím se ještě podívat na `rofi_hal.hpp`.
Nemám příležitost si to vyzkoušet, ale přijde mi to v pořádku (nevím jak `hal`).

File: .github/workflows/build.yml [hunk 5]
```diff
@@ -24,6 +24,27 @@ jobs:
             source setup.sh ${{ matrix.environment }}
             rcfg desktop
             rmake desktop
+      - name: Build RoFI firmware
+        uses: addnab/docker-run-action@v2
+        with:
+          image: yaqwsx/rofi.desktop
```
[the remark this answers is not in the document]
Comment (yaqwsx) [reply .github/workflows/build.yml:48]: Nn, všechno se asi postupně zmigruje do jednoho obrazu - do dektopu už totiž jen chybí GCC pro ARM.

**Pull request merged**